src/radio/index.tsx

```diff
@@ -46,125 +45,114 @@ export interface RadioProperties extends ThemedProperties, FocusProperties {
 	widgetId?: string;
 }
 
-@theme(css)
-export class Radio extends ThemedMixin(FocusMixin(WidgetBase))<RadioProperties> {
-	private _uuid = uuid();
-
-	private _onChange(event: Event) {
-		event.stopPropagation();
-		const radio = event.target as HTMLInputElement;
-		this.properties.onValue && this.properties.onValue(radio.checked);
-	}
+const factory = create({ focus, theme }).properties<RadioProperties>();
 
-	protected getRootClasses(): (string | null)[] {
-		const { checked = false, disabled, valid, readOnly, required } = this.properties;
-		const focus = this.meta(Focus).get('root');
+export const Radio = factory(function Radio({ properties, middleware: { focus, theme } }) {
+	const _uuid = uuid();
```

Comment:
can you use the `id` from  the factory instead of `uuid` please and prefix with `radio`, similar to https://github.com/dojo/widgets/blob/master/src/menu/index.tsx#L291